### DISCUSSION TITLE
Unity XR Hands integration

### DISCUSSION THREAD

**Discussion by bdovaz** [leapmotion/UnityModules#1419] [General]
This package has recently become stable with the release of XR Interaction toolkit 2.3.0.

It allows accessing hand information from an OpenXR-like abstraction.

https://docs.unity3d.com/Packages/com.unity.xr.hands@1.1/manual/implement-a-provider.html

It would be nice if you could implement it so we don't have to use any Leap specific API and swap between different hardware without having to change the code.

**Comment by MattGrayUL** [comment]
Hi @bdovaz

We have plans to do just this, across the Plugin for XR use-cases. This is not a small undertaking as all of our features are based on the existing LeapCSharp structures, and so will take some time

We are working closely with Unity to ensure this is as suitable as a replacement as possible for our Plugin and for the interactions we recommend

Thanks for the mention though, it really helps to direct the focus of the team 🙂

As this is not necessarily an existing _issue_, I have converted this into a discussion 🙂
